client/images/icons/actions/print.svg

```diff
@@ -0,0 +1,23 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 22 22">
+  <defs>
+    <style>
+      .cls-1,
+      .cls-2 {
+        fill:none;
+        stroke-linecap:round;
+        stroke-width:2px;
+      }
+      .cls-1 {
+        stroke-miterlimit:10;
+      }
+      .cls-2 {
+        stroke-linejoin:round;
+      }
+    </style>
+  </defs>
+  <path class="cls-1 icon-sprite__shape" stroke="#1d1d1d" d="M18,17a2,2,0,0,0,2-2V10.5C20,9.4,19.1,8,18,8H4C2.89,8,2,9.4,2,10.5V15a2,2,0,0,0,2,2Z"/>
+  <rect class="cls-2 icon-sprite__shape" stroke="#1d1d1d" x="6" y="2" width="10" height="6"/>
+  <rect class="cls-2 icon-sprite__shape" stroke="#1d1d1d" x="6" y="17" width="10" height="3"/>
+  <line class="cls-1 icon-sprite__shape" stroke="#1d1d1d" x1="6" y1="11" x2="7" y2="11"/>
+  <line class="cls-1 icon-sprite__shape" stroke="#1d1d1d" x1="10" y1="11" x2="11" y2="11"/>
+</svg>
```

Comment:
i did remove unnecessary attributes and only needed to bring in the stroke value as an attribute to enable override with CSS. It think it makes sense to keep the other styles. Everything gets minified in the icon-sprite